_posts/2019-02-15-introducing-aquery.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: posts
+title: Introducing Bazel Aquery
+authors:
+ - joeleba
+---
+
+_tl;dr: [`bazel aquery`](https://docs.bazel.build/versions/master/aquery.html) is a new bazel command that queries the action graph, and thus allows you to gain insights about the actions executed in a build (inputs, outputs, command line, …).
+`aquery`’s API is now stable and supported by the Bazel team._
+
+## Background & Motivation
```

Comment:
I think you should start with a specific example of a user problem that `aquery` can help solve, and then list a class of problems that aquery solves. That will help your readers to be motivated to read through the rest of the post.

Reply:
Resolved in doc.